src/utils/options.ts

```diff
@@ -159,13 +184,13 @@ export default class Options {
   }
 
   public getUserHomeDir(): string {
-    if (this.isPortable()) return process.env.VSCODE_PORTABLE as string;
+    if (this.isPortable()) return process.env.WAKATIME_DESKTOP_PORTABLE as string;
```

Comment:
just a question. Was `VSCODE_PORTABLE` constant changed to `WAKATIME_DESKTOP_PORTABLE`?

Reply:
yes when i did this change i was thinking that i does not make sense to use a reference to vscode, since this is the desktop app